pkgs/desktops/gnome-3/core/evince/default.nix

```diff
@@ -66,13 +84,30 @@ stdenv.mkDerivation rec {
     libsecret
     librsvg
     adwaita-icon-theme
+    dbus
```

Comment:
What do we need dbus for?

Reply:
IIRC evince instances can be controlled via dbus, e.g. to move to a particular point in a document, like synctex.  There is also a check for dbus in the meson.build file, I only added it to buildInputs because the build failed otherwise.

Reply:
`glib` based apps usually rely on its `gdbus` implementation instead of `libdbus`. I would only expect `dbus-1` to be needed for finding the service directory. And apparently it [is](https://gitlab.gnome.org/GNOME/evince/blob/78a97647a293bdb989b8419bb301f87298a89ca8/meson.build#L185). Can we add a comment stating so?

Reply:
Added comment.